projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/datajobs/it/common/JobExecutionUtil.java

```diff
@@ -150,7 +150,7 @@ public static ImmutablePair<String, String> executeDataJob(
     return ImmutablePair.of(opId, executionId);
   }
 
-  private static void testDataJobExecutionRead(
+  public static void testDataJobExecutionRead(
```

Comment:
Make methods public. 
They are clean and I think it makes sense for them to be public. I need them in the test class above.